readme.md

```diff
@@ -48,6 +48,44 @@ await getData("https://open.spotify.com/track/5nTtCOCds6I0PHMNtqelas");
 
 returns any raw data we can scrape from spotify. There are no guarantees about the shape of this data, because it varies with different media and scraping methods. Handle it carefully.
 
+
+```javascript
+await getTitles("https://open.spotify.com/playlist/3Q4cPwMHY95ZHXtmcU2xvH");
+```
+
+returns array with tracks. Bellow is array with one track with guaranteed properties.
```

Comment:
Bellow -> Below

Reply:
> Bellow -> Below

I had there below but it seemed weird. English is my second language. I'm from czech republic

Reply:
Yeah, i feel ya. I'm german so I always have to double check that my Capitalization is Not all Over the Place.